osmomath/decimal.go

```diff
@@ -552,23 +560,45 @@ func (d BigDec) MustFloat64() float64 {
 // Dec returns the osmomath.Dec representation of a BigDec.
 // Values in any additional decimal places are truncated.
 func (d BigDec) Dec() Dec {
-	precisionDiff := PrecisionBigDec - PrecisionDec
-	precisionFactor := new(big.Int).Exp(big.NewInt(10), big.NewInt(int64(precisionDiff)), nil)
+	return d.DecWithPrecision(PrecisionDec)
+}
```

Comment:
Do you mind also adding `DecMut` please?

See the comments in https://github.com/osmosis-labs/osmosis/pull/6369

Reply:
This could be done in a separate PR. I'm just bumping it up since you already have context and might be interested

Reply:
opened a PR: https://github.com/osmosis-labs/osmosis/pull/6393